acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/Eth2BlockSigningAcceptanceTest.java

```diff
@@ -88,14 +88,18 @@ void signAndVerifyLegacyBlockSignature() throws Exception {
     assertThat(signature).isEqualTo(expectedSignature.toBytesCompressed());
   }
 
-  @Test
-  void emptyBlockRequestReturnsBadRequestStatus() throws JsonProcessingException {
-    final Eth2BlockSigningRequestUtil util =
-        new Eth2BlockSigningRequestUtil(SpecMilestone.BELLATRIX);
-    setupEth2Signer(Eth2Network.MINIMAL, SpecMilestone.BELLATRIX);
+  @ParameterizedTest(
+      name = "#{index} - Empty block request for spec {0} should return bad request status")
+  @EnumSource(
+      value = SpecMilestone.class,
+      names = {"BELLATRIX", "CAPELLA"})
```

Comment:
Why does this only apply to bellatrix and capella? should also include phase0 and altair?

Reply:
good point, updated it to cover phase0 and altair as well.